indico/modules/events/papers/operations.py

```diff
@@ -130,32 +130,36 @@ def close_cfp(event):
     event.log(EventLogRealm.management, EventLogKind.negative, 'Papers', 'Call for papers closed', session.user)
 
 
-@no_autoflush
-def create_paper_revision(contribution, submitter, files):
-    paper = Paper(contribution=contribution)
+def create_paper(contribution):
```

Comment:
creating just a model instance but not adding it anywhere or doing anything else that's not trivial is not an operation IMO